src/annotator/components/test/NotebookModal-test.js

```diff
@@ -58,6 +58,9 @@ describe('NotebookModal', () => {
     );
   });
 
+  // Test disabled temporarily until a sync mechanism is implemented in the
```

Comment:
I think it's still valid to leave this test even though it's now not possible to change the groupid and _not_ the iframeKey.  

**OR** I would suggesting getting rid of the `key` on the iframe too and just removing all the code bits and this test too.

Reply:
I removed the `key` from the iframe and re-enable the test adjusting to the new behaviour.